CHANGELOG.md

```diff
@@ -5,7 +5,9 @@
 ## 20.30.2 - 2023-09-13
 
 ### PaymentSheet
-* [ADDED][7281](https://github.com/stripe/stripe-android/pull/7281) PaymentSheet now supports OXXO for PaymentIntents.
+* [ADDED] PaymentSheet now supports the following payment methods for PaymentIntents:
+  * [7281](https://github.com/stripe/stripe-android/pull/7281) OXXO
+  * [7302](https://github.com/stripe/stripe-android/pull/7302) Alma
```

Comment:
Looks like this is in a previous release again (sorry for the hotfix).

Can you bump this up and make it look something like 

```
### PaymentSheet
* [ADDED][7302](https://github.com/stripe/stripe-android/pull/7302) PaymentSheet now supports Alma for PaymentIntents.

```

But now that I'm thinking about this, I'm not sure we want to do this if merchants need to be gated. Do we have a link where they can request an invitation to the beta?